tests/robot-tests/tests/admin_and_public/bau/publish_methodology.robot

```diff
@@ -415,7 +415,12 @@ Verify that the amended methodology is visible on the public methodologies page
     user scrolls down    400
     user checks page contains link with text and url
     ...    ${PUBLICATION_NAME} - Amended methodology
-    ...    ${PUBLIC_METHODOLOGY_URL_ENDING}
+    ...    ${PUBLIC_METHODOLOGY_URL_ENDING}-amended-methodology    # Slug has changed
+
+Validate methodology redirect works
```

Comment:
@mmyoungman this test will need to wait for the redirect cache to expire. It's set to 10 seconds on local and dev so we might want to add WAIT_REDIRECT_CACHE_EXPIRY variable and a keyword for it.